alacritty_terminal/src/term/mod.rs

```diff
@@ -124,6 +138,16 @@ impl<'a> RenderableSearch<'a> {
     }
 }
 
+/// Common rendering primitives stored in one structure to simplify passing them around.
+struct RenderableContentInner<'a, T, C> {
```

Comment:
I have no clue how to name it, maybe `RenderableContext`.

Reply:
`RenderContext` maybe? `RenderableContext` would mean it's a context that can be rendered.